tests/src/rules/no-import-module-exports.js

```diff
@@ -93,16 +93,6 @@ ruleTester.run('no-import-module-exports', rule, {
         import foo from 'path';
         module.exports = foo;
       `,
-      // When the file does not match the entry point defined in package.json
-      // See tests/files/package.json
```

Comment:
Ahh....the four tests above this one are already testing that the rule applies for files that aren't "main". By default the `test` util sets the default file to `eslint-plugin-import/tests/files/foo.js`. Which isn't equal to the "main" I set up in `eslint-plugin-import/tests/files/package.json`. It seemed a bit redundant to me...but I can add it back if you prefer.